other/shuffled_array.py

```diff
@@ -0,0 +1,71 @@
+# To create a shuffled array. Generate a pseudo random number
+# Swap that number with any other element in the list run this swapping len(Array)
+# times to get desired shuffled array
+
+# To Understand more of random number generation
+# follow https://en.wikipedia.org/wiki/Lagged_Fibonacci_generator
+
+# import doctest
+import time
+
+
+class Solution:
+
+    # """
+    # >>> Solution().__init__(array=[1,2,3])
+    # Traceback (most recent call last):
+    #            ....
+    # TypeError: __init__() missing 1 required positional argument: 'array'
+    # """
+
+    def __init__(self, array: list) -> None:
+        self.arr = array
+        self.seed = int(str(time.time())[-1:-5:-1])
+
+    # generating a 4 digit number randomly
+    # by taking the last four numbers of the system generated time
+    # pseudo random number generator
+    def prng(self, num: int) -> int:
+        """
+        >>> Solution([56]).prng(1)
+        0
+        """
+        if num == 1:
+            return 0
+        self.seed *= self.seed
+        any = str(self.seed)
+        if any != "0":
+            self.seed = int(any[-1:-5:-1])
+        else:
+            any = any[:] + str(len(self.arr) // 2)
+        if int(any[-1]) < num:
+            return int(any[-1])
+        return self.prng(num)
+
+    def reset(self) -> list:
+        # it will return the original given array
+        """
+        >>> Solution([2,3,4,56]).reset()
+        [2, 3, 4, 56]
```

Comment:
Let's test an unsorted sequence that includes negative ints, zero, and floating point numbers.
```suggestion
        >>> Solution([-7, 0, 4, -56.7]).reset()
        [-7, 0, 4, -56.7]
```